packages/ionic-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nxext/ionic-react",
-  "version": "17.0.0",
+  "version": "18.0.4",
```

Comment:
@eduardoRoth @DominikPieper I bumped the version manually as I didn't get how the release works :wink:

Reply:
Yes that’s fine

Reply:
@DominikPieper feel free to merge it :)